unicon_backend/lib/permissions/__init__.py

```diff
@@ -44,7 +91,8 @@ def debug_list_tuples():
 
 
 def delete_all_permission_records():
-    _data_api.data_delete_without_preload_content(PERMIFY_TENANT_ID, p.DataDeleteBody())
+    tuples, attributes = _get_all_tuples_and_attributes()
```

Comment:
just curious, what is the difference between the previous `.data_delete_without_preload_content` and the current way of first querying for tuples and attributes and then manually deleting it?

Reply:
the previous one never worked and returned an error (tuple/attribute filters are missing), so seed was broken